### DISCUSSION TITLE
Navigation TOC Animation

### DISCUSSION THREAD

**Discussion by blogi-kriptovalyut** [mmistakes/minimal-mistakes#2671] [Q&A]
Hey. I may have found a bug.
If the site page is in English, then when you click on the navigation TOC, the page scrolls smoothly.
If the site page is in Russian, then when you click on the navigation TOC, the page scrolls abruptly (no animation).

- Minimal Mistakes version: 4.20.1
- Ruby gem or remote theme version: 2.6.6
- Jekyll version: 4.1.1
![img-2020-08-28-10-41-07](https://user-images.githubusercontent.com/50487083/91534904-01b13180-e91b-11ea-96bd-8e5389440b19.png)

- Operating system: win 10

**Comment by iBug** [comment]
Doesn't look right to me. Have you tried disabling all those fancy stuff that don't come with the theme and trying again?

**Comment by blogi-kriptovalyut** [comment]
> Doesn't look right to me. Have you tried disabling all those fancy stuff that don't come with the theme and trying again?

Yes, it works on a clean theme as intended. I checked. I don't understand what could have broken the animation...

**Comment by blogi-kriptovalyut** [comment]
@mmistakes, please tell me which files are responsible for animation in TOC navigation?

**Comment by mmistakes** [comment]
@blogi-kriptovalyut There isn't any animation specific to the TOC navigation. The site uses a [smooth scroll JavaScript](https://github.com/mmistakes/minimal-mistakes/blob/master/assets/js/plugins/smooth-scroll.js) that smoothly scrolls to internal page links.

My guess is you added something that broke that script when customizing things.
